velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -41,6 +41,15 @@ inline void registerArrayJoinFunctions() {
       Varchar>({"array_join"});
 }
 
+template <typename T>
+inline void registerArrayCombinationsFunctions() {
+  registerFunction<
+      ParameterBinder<CombinationsFunction, T>,
+      Array<Array<T>>,
+      Array<T>,
+      int64_t>({"combinations"});
```

Comment:
@laithsakka @bikramSingh91 In Presto, the type of second parameter is INTEGER, not BIGINT.

```
presto:di> show functions like 'combinations';
   Function   |   Return Type   |  Argument Types   | Function Type | Deterministic |                Description                | Variable Arity | Built In | Temporary | Language
--------------+-----------------+-------------------+---------------+---------------+-------------------------------------------+----------------+----------+-----------+----------
 combinations | array(array(T)) | array(T), integer | scalar        | true          | Returns n-element combinations from array | false          | true     | false     |
(1 row)
```